File: Implementation/Python/GCD.py
def greatest_common_divider(a, b):
    while a != b:
        if a > b:
            a  =  a - b
        else:
            b = b - a
    return a

def main():
    a = 20 
    b = 50
    print(greatest_common_divider(a, b))

if __name__ == "__main__":
    main()
 

File: Implementation/Other/C++/GCD.cpp
#include <iostream> 

using namespace std;

int greatest_common_divider(int a, int b)
{
    while(a != b)
    {
        if (a > b)
            a = a - b;
        else 
            b = b - a;
    }
    return a;
}

int main()
{
    int a = 20;
    int b = 50;
    cout << greatest_common_divider(a, b) << "\n";
}